src/test/components/MenuButtons.test.js

```diff
@@ -63,8 +63,20 @@ const { UploadAbortedError } = jest.requireActual(
 import sha1 from '../../utils/sha1';
 jest.mock('../../utils/sha1');
 
-// Mocking compress
-jest.mock('../../utils/gz');
```

Comment:
This was mocked previously because a long time ago this module didn't work. But because of that we weren't testing the code correctly. Indeed `compress` was returning `undefined` instead of an object of `Promise<Uint8Array>`, triggering a few side effects.

For example we were still creating a Blob out of this, and as a result this Blob contained the string `"undefined"`, that's why you'll see in the snapshots that the size was 9 bytes previously. Now it's a more sensible value.

Another consequence is that before the Promise was resolved already, but now it needs to go through the Worker to compress the data. That's why we have to use some `findBy` functions to wait until the compression ends.